packs/nitropack.js

```diff
@@ -0,0 +1,41 @@
+
+const UIStrings = {
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Reduce Unused CSS` is the name of a feature and becomes link text to additional documentation.*/
+  'unused-css-rules': 'Enable [`Reduce Unused CSS`](https://support.nitropack.io/hc/en-us/articles/360020418457-Reduce-Unused-CSS) to remove CSS rules that are not applicable to this page. This directly affects how fast the browser builds your page.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Image Optimization` is the name of a feature and becomes link text to additional documentation. `WebP` is the name of a standardized image format for the web.*/
+  'modern-image-formats': 'Use [`Image Optimization`](https://support.nitropack.io/hc/en-us/articles/16547237162513) to automatically convert your images to WebP and improve your loading times.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Automatic Image Lazy Loading` is the name of a feature and becomes link text to additional documentation.*/
+  'offscreen-images': 'Defer offscreen images by enabling [`Automatic Image Lazy Loading`](https://support.nitropack.io/hc/en-us/articles/12457493524369-NitroPack-Lazy-Loading-Feature-for-Images). This will load images only when they need to be displayed on user’s screen, resulting in quicker page load times.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Image Optimization`, `Minify HTML`, `Font Subsetting`, `Reduce Unused CSS` are names of features and become links to additional documentation.*/
+  'total-byte-weight': 'To reduce network payloads, enable the following NitroPack settings that help with the compression and lazy loading of your resources, including — [`Image Optimization`](https://support.nitropack.io/hc/en-us/articles/16547237162513); [`Minify HTML`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources); [`Font Subsetting`](https://support.nitropack.io/hc/en-us/articles/10172273306769); [`Reduce Unused CSS`](https://support.nitropack.io/hc/en-us/articles/360020418457-Reduce-Unused-CSS); [`Font Compression Upgrade`](https://support.nitropack.io/hc/en-us/articles/16547272270865).',
```

Comment:
I would suggest removing this suggestion since it's just repeating suggestions from other audits.

```suggestion
```